include/dlaf/solver/triangular/impl.h

```diff
@@ -425,10 +425,48 @@ void Triangular<backend, device, T>::call_LLN(comm::CommunicatorGrid grid, blas:
   }
 }
 
-template <Backend backend, Device device, class T>
-void Triangular<backend, device, T>::call_LLT(comm::CommunicatorGrid grid, blas::Op op, blas::Diag diag,
-                                              T alpha, Matrix<const T, device>& mat_a,
-                                              Matrix<T, device>& mat_b) {
+template <Backend B>
+struct getGenericExecutor {
+  static auto call() {
+    return dlaf::getNpExecutor<Backend::MC>();
+  }
+};
+
+#ifdef DLAF_WITH_CUDA
+template <>
+struct getGenericExecutor<Backend::GPU> {
+  static auto call() {
+    return dlaf::cuda::Executor{dlaf::internal::getNpCudaStreamPool()};
+  }
+};
+#endif
```

Comment:
:+1: 

Perhaps a comment saying why this is here instead of in `executors.h`?

Reply:
Is it correct saying that it is a workaround because CuSolver/CuBlas executors do not use the basic CUDA executor as fallback?

Reply:
Yep, that's accurate!